src/pages/contributors/vincent-latombe.adoc

```diff
@@ -0,0 +1,137 @@
+Later on, when CloudBees did some fundraising, they started to hire some of the community members.
+Kohsuke offered me a position at CloudBees, which I accepted.
+
+== How long have you been using Jenkins?
+
+I started using Jenkins in the late 2000's when it was still under the Hudson name.
+I eventually started contributing to Jenkins core around 2010 or 2011, along with submitting fixes to the Clear Case plugin.
+Now, I've been working on Jenkins for the last ten years and have seen it grow in many ways since my initial usage.
+ 
+== Why choose Jenkins over other projects?
+
+As far as Jenkins is concerned, I've always been impressed by how versatile it is as a solution.
+Before using Jenkins, every time we wanted to change some configuration, we needed to apply a restart.
+The restart would cancel the builds and everything was much more complicated than it needed to be.
+With Jenkins, you could set up new jobs and adjust configuration for each one.
+You didn't need to restart in order to apply your changes.
+Jenkins removed a lot of the headaches we would deal with when it came to configuring and running builds.
+
+== What problems has Jenkins solved for you?
+
+In my previous job before CloudBees, we were using either Maven jobs or freestyle jobs.
+We started to consider building a project with different branches, where a job uses the branch as a parameter.
+This isn't ideal, because the different builds you get are unrelated to each other.
+So the best structure was to create one job per branch.
+However, it messy and complicated because there was no way to capture what was common between the different jobs.
+Obviously, at the time we did not have Pipeline.
+Therefore, we chose to build an automation using a template that would copy it over, customize the branch name, and then repeat that for the dozens of projects that were following the same lifecycle but managed as different projects.
+All of these projects needed to be managed and we needed to have a way to determine whether a component actually changed, otherwise a build that comes from the main branch would be reused.
+Every time we had a big project, it became very costly if we didn't do it in a smart way.
+So we always try to optimize for the infrastructure and time because otherwise we pay for the computing processes, storage, network, and other aspects.
+Jenkins helped with this optimization and provided a way for us to manage all the jobs that were being run.
+
+== What have you worked on in Jenkins?
+
+Initially, when I got recruited at CloudBees, they were building proprietary plugins on top of Jenkins as well as contributing to the community.
+It was the very beginning of cloud support, which eventually became CI modern.
+It was basically a prototype, where we were trying to generalize how to provision Jenkins instances automatically without things like Cask, Docker, and a lot of other things that were new at the time.
```

Comment:
```suggestion
When I came in, I started to work on a prototype of a deployment of Jenkins in the cloud.
We were trying to generalize how some companies were doing massive deployments of Jenkins on clustered systems like Mesos, but packaged in a product that could be run by any of our customers.
At the time, we started using Mesos and Marathon, while using Docker as a container engine.
Since we were running on AWS (which was still charging by the hour), and Docker volumes did not exist, we had to create our own system to allocate ebs volumes on the fly.
```